package.json

```diff
@@ -7,8 +7,10 @@
     "ionicons": "^3.0.0",
     "react": "16.2.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-25.0.0.tar.gz",
+    "react-native-elements": "^0.19.0",
     "react-native-router-flux": "^4.0.0-beta.28",
     "react-native-vector-icons": "^4.5.0",
+    "react-navigation": "^1.0.0-beta.27",
```

Comment:
Now using `react-navigation` (we'll remove `react-native-router-flux` entirely once the refactor is complete), and added `react-native-elements` which has been a great quick-win.